store/models.py

```diff
@@ -570,14 +570,19 @@ class OptionProduct(BaseProduct):
         help_text=_("Maximum allowed quantity per orderline"),
         default=0
     )
-    has_stock = models.BooleanField(
-        verbose_name=_("Has stock"),
-        help_text=_("True if option has stock, False if stock is infinite or NA"),
+    manage_stock = models.BooleanField(
+        verbose_name=_("Manage stock"),
+        help_text=_("True if option manage stock, False if stock is infinite or NA"),
         default=False
     )
     stock = models.PositiveIntegerField(
         verbose_name=_("Stock"),
-        help_text=_("Maximum quantity available for this option"),
+        help_text=_("Current quantity available for this option"),
+        default=0
+    )
+    initial_stock = models.PositiveIntegerField(
+        verbose_name=_("Initial stock"),
+        help_text=_("Initial stock for this option"),
         default=0
     )
```

Comment:
This architecture seems a bit dangerous for me:

 - If admin want to upgrade the stock, he will need to update both field to keep things clean
 - We have a copy of information, and it's not a good principles in SQL schema

If we want to have initial stock in DB, the actual stock could be just calculated everytime we need it with a simple DB request.